### PR TITLE
[UIA-42] Imported MendixSSO v3.0.0

### PR DESCRIPTION
**Pull request by MendixMurshed**
UIA-42 [MxSSO] Update templates with MxSSO and Administration module and using the connector logic 

We have imported MendixSSO to this template, this is only the first step of updating this template and will follow later with enabling MendixSSO by default.

### CODE REVIEW

**Review by KevinVlaanderen — CHANGES_REQUESTED** [review]
I don't think NanoflowCommons should be changed here

File: theme-cache/web/theme.compiled.css [hunk 1]
```diff
@@ -21963,6 +21963,7 @@ html div.dj_ios .widget-switch.auto .widget-switch-btn-wrapper.checked.widget-sw
   margin-left: 8px;
   flex-grow: 0;
   color: #606671;
+  height: 14px;
```
Comment (KevinVlaanderen) [inline theme-cache/web/theme.compiled.css:21966]: Should the theme-cache be committed?

**Review by KevinVlaanderen — CHANGES_REQUESTED** [review]

**Comment by diego-antonelli** [comment]
@KevinVlaanderen @allard-mx @MendixMurshed We usually do not do PRs for mpr changes (model) as they are untraceable. And now is causing merge conflict.

**Comment by KevinVlaanderen** [comment]
> 

Right. What's the preferred way to request changes in the starter apps?

**Comment by diego-antonelli** [comment]
> I don't think NanoflowCommons should be changed here

The version in Blank app is pushed before the compile, so every time the app runs it changes the format of export (instead of export {jsaction} the export is moved to the function)

**Comment by diego-antonelli** [comment]
> > 
> 
> Right. What's the preferred way to request changes in the starter apps?

Push directly to main branch

File: theme-cache/web/theme.compiled.css (hunk 1 above)
Comment (diego-antonelli) [reply theme-cache/web/theme.compiled.css:21966]: There was a previous problem in Studio requiring theme-cache, thats why its not ignored in the .gitignore

**Pull request closed without merging**